src/screens/Ticketing/Ticket/CompactTicketInfo.tsx

```diff
@@ -148,9 +165,21 @@ export const getTicketInfoTexts = (
     conjunction,
     serialComma: false,
   });
-  const timeUntilExpire = t(TicketTexts.validityHeader.valid(durationText));
-
-  var accessibilityLabel = timeUntilExpire + screenReaderPause;
+  var timeUntilExpireOrInvalid: string;
+
+  if (isMobileToken(inspectableToken) && !isInspectable) {
```

Comment:
This isn't ment to be showed if t:card is the inspectable token? 🤔

Reply:
The accessibility changes when the token is valid or inspectable on another device. These conditions evaluate that, but do you see something is missing? Please let me know the logic about tickets is a quite complex to follow 😅

Reply:
It was not regarding accessibility label, but regarding the `timeUntilExpireOrInvalid` value. By just reading it, it seems that if the inspectable token is a t:card, then the normal validity header "Expires in ..." is shown. I don't know if this is intended or not.

Reply:
Hey! After our talk I made improvements in the overall code, now it looks slightly better and fixed with the invalid ticket!